src/beamlime/frontend/matplotlib_flask.py

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+"""
+Demo of a simple Flask app that plots live data from a ZMQ server using Matplotlib.
+"""
+
+import asyncio
+import os
+import threading
+from collections import deque
+from io import BytesIO
+
+import msgpack
+from flask import Flask, Response, render_template_string, request
+from matplotlib.figure import Figure
```

Comment:
We need to force matplotlib to pick a non interactive backend with `matplotlib.use('agg')` https://github.com/matplotlib/matplotlib/issues/14304/#issuecomment-545717061

Reply:
Fixed.